src/Microsoft.AspNetCore.Http.Extensions/RequestHeaders.cs

```diff
@@ -260,6 +277,23 @@ public RangeHeaderValue Range
                 Headers.Set(HeaderNames.Range, value);
             }
         }
+        
+        public Uri Referer
```

Comment:
Referer spec for reference: https://tools.ietf.org/html/rfc7231#section-5.5.2

Reply:
Looks good. Nothing sticking out in the specification.